sheetstorm/tests/test_views.py

```diff
@@ -0,0 +1,23 @@
+from django.conf import settings
+from django.test import TestCase
+from django.urls import reverse
+
+from sheetstorm.views import Index
+from users.factories import UserFactory
+
+
+class IndexPageTests(TestCase):
+    def setUp(self):
+        self.url = reverse("home")
+        self.user = UserFactory()
+
+    def test_not_logged_user_got_redirected_to_login_page(self):
```

Comment:
Isn't that testing framework functionality?

Reply:
But these are simple 2 unit tests to be sure that we have access to home page :)

Reply:
Lets keep it then, but please also add to test `test_access_permissions`.

Reply:
Also added :)